src/V3Graph.h

```diff
@@ -182,6 +183,38 @@ class V3Graph VL_NOT_FINAL {
     void dumpEdges(std::ostream& os, const V3GraphVertex* vertexp) const VL_MT_DISABLED;
     static void selfTest() VL_MT_DISABLED;
 
+    class ParallelismReport final {
+        friend class GraphAlgParallelismReport;
+        // Total cost of evaluating the whole graph. The ratio of m_totalGraphCost to
+        // m_criticalPathCost gives us an estimate of the parallelizability of this graph which is
+        // only as good as the guess returned by vertexCost.
+        uint32_t m_totalGraphCost = 0;
```

Comment:
I realize this is code movement, so fine to commit, but afterwards please commit a change to change this to uint64_t s (and maybe some of the others) as I think this can easily overflow..  (Realistically we're limited to ~ 2^40 for 1TB memory / 2 ^ 6 byte vertex = 2 ^ 32 vertices, so even if average weight only was 1 we could break.)

Reply:
Yep, will do.